### ISSUE TITLE
Add "soft thresholding" post processing

### ISSUE THREAD

**Issue by lrouhier** [ivadomed/ivadomed#463]
It would be useful to have a soft thresholding/ denoising function for soft output. 
The idea would be to put all value below a certain threshold to 0 and kept the other one. This would allow us to clean soft output if needed. 
See example below : 
![image](https://user-images.githubusercontent.com/22171999/94858653-5efc3f00-0401-11eb-927c-cebf56ef8c68.png)

**Comment by charleygros** [comment]
@lrouhier : I think that what you are looking for is essentially this: [here](https://github.com/ivadomed/ivadomed/blob/master/ivadomed/utils.py#L115) --> we would need to adjust this value via a post-processing parameter instead of hardcoding.

**Comment by lrouhier** [comment]
You are right, we could use value 0 to have the same behavior as "denoise = False" . I will look into that :)
